docs/network/index.md

```diff
@@ -29,7 +29,7 @@ Overview of Celo Mainnet, Alfajores L2 Testnet and Baklava Testnet.
 | Currency Symbol            | CELO                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    |
 | Chain ID                   | 44787                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
 | RPC Endpoint (best effort) | [https://alfajores.celoscan.io/](https://alfajores.celoscan.io/g)                                                                                                                                                                                                                                                                                                                                                                                                  |
-| RPC Nodes                  | <ul><li>Ethereum JSON-RPC endpoint: [https://alfajores.celoscan.io/g](https://alfajores.celoscan.io/) (op-geth kind)</li><li>OP RPC endpoint: [https://op.alfajores.celo-testnet.org/](https://op.alfajores.celo-testnet.org/), (op-node kind)OP RPC endpoint: [https://op.alfajores.celo-testnet.org/](https://op.alfajores.celo-testnet.org/) (op-node kind)</li><li>Infura: [https://www.infura.io/networks/celo](https://www.infura.io/networks/celo)</li></ul> |
+| RPC Nodes                  | <ul><li>Open RPC Endpoint (Forno): [https://alfajores-forno.celo-testnet.org/](https://alfajores-forno.celo-testnet.org/)</li><li>Infura: [https://www.infura.io/networks/celo](https://www.infura.io/networks/celo)</li><br/><li>Rollup Endpoint: [https://op.alfajores.celo-testnet.org/](https://op.alfajores.celo-testnet.org/))</li></ul> |
```

Comment:
What is the rollup endpoint ?

Reply:
https://docs.optimism.io/operators/node-operators/json-rpc